graphql_api/types/repository/repository.py

```diff
@@ -24,6 +25,8 @@
 from services.profiling import CriticalFile, ProfilingSummary
 from services.redis_configuration import get_redis_connection
 
+TOKEN_UNAVAILABLE = "Token Unavailable"
```

Comment:
Is the plan for UI just to show this instead of the token? Or check the value and do something special?

Reply:
Just updated the PR and copy. Since this work is just for a single customer, we are just going to render the value directly.

Reply:
Cool sounds good. I prefer this approach given this is not an official feature.